grassroots-frontend/src/routes/CreateContact.tsx

```diff
@@ -25,6 +25,7 @@ function CreateContact(): JSX.Element {
     mutationFn: async (contact: CreateContactInDto) => {
       const result = await grassrootsAPI.POST("/contacts", {
         body: contact,
+        credentials: "include",
```

Comment:
Will we need to do this on every route?

Reply:
We actually don't need it on any route: cookies are included in same-origin requests by default. Removed.